server/api/team/tests.py

```diff
@@ -0,0 +1,3 @@
+# from django.test import TestCase
+
+# Create your tests here.
```

Comment:
please learn how to do testing see 
here's an example https://www.django-rest-framework.org/api-guide/testing/#api-test-cases

Reply:
Dont worry about testing - we can create a new issues for this :)

Reply:
Then create a new issue Chantelle!